intel_extension_for_transformers/backends/neural_engine/graph/jblas/CMakeLists.txt

```diff
@@ -24,5 +29,5 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 include(FindOpenMP)
 target_link_libraries(${PROJECT_NAME} INTERFACE OpenMP::OpenMP_CXX OpenMP::OpenMP_C)
-target_compile_features(${PROJECT_NAME} INTERFACE cxx_std_11)
+target_compile_features(${PROJECT_NAME} INTERFACE cxx_std_17)
```

Comment:
Is it problematic if a head-only library uses newer standard then that the project depends on it?

cc @airMeng

Reply:
cxx17 is not that new, only gcc>=7 should be alright. BTW, cxx26 is under development.